stripe/src/main/java/com/stripe/android/IssuingCardPinService.java

```diff
@@ -0,0 +1,290 @@
+package com.stripe.android;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.VisibleForTesting;
+
+import com.stripe.android.exception.APIConnectionException;
+import com.stripe.android.exception.APIException;
+import com.stripe.android.exception.AuthenticationException;
+import com.stripe.android.exception.CardException;
+import com.stripe.android.exception.InvalidRequestException;
+
+import org.json.JSONException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/*
+ * Methods for retrieval / update of a Stripe Issuing card
+ */
+public class IssuingCardPinService
+        implements EphemeralKeyManager.KeyManagerListener<IssuingCardEphemeralKey> {
+
+    private static final long KEY_REFRESH_BUFFER_IN_SECONDS = 30L;
+    private static final String PIN_RETRIEVE = "PIN_RETRIEVE";
+    private static final String PIN_UPDATE = "PIN_UPDATE";
+    private static final String ARGUMENT_CARD_ID = "cardId";
+    private static final String ARGUMENT_VERIFICATION_ID = "verificationId";
+    private static final String ARGUMENT_ONE_TIME_CODE = "userOneTimeCode";
+    private static final String ARGUMENT_NEW_PIN = "newPin";
+    @NonNull
```

Comment:
line break between the static and instance fields

Reply:
👍